src/OrchardCore/OrchardCore.ResourceManagement/ResourceManagementConstants.cs

```diff
@@ -0,0 +1,8 @@
+namespace OrchardCore.ResourceManagement;
+
+internal static class ResourceManagementConstants
+{
+    public static readonly char[] ParameterValueSeparator = [',', ' '];
```

Comment:
```suggestion
    public static readonly char[] ValuesSeparator = [',', ' '];
```

Reply:
I don't think the separator for a value, or you could use `ParameterValuesSeparator`

Reply:
I don't think I understand what you are asking here. `ParameterValueSeparator` is better than `ValuesSeparator` because these are specific value separator.

Reply:
done